src/Game/Client.java

```diff
@@ -454,11 +480,36 @@ public static void init() {
     // Initialize login
     init_login();
 
-    // Skip first login screen and don't wipe user info
-    login_screen = 2;
-
     init_extra();
   }
+  
+  public static boolean skipToLogin() {
+	  boolean skipToLogin = false;
+	  
+	  if (Settings.noWorldsConfigured || (Settings.WORLDS_TO_DISPLAY == 1 && Settings.WORLD.get(Settings.currentProfile) != 0)) {
+	    	String curWorldURL = Settings.WORLD_URLS.get(1);
+	    	try {
+				String address = InetAddress.getByName(curWorldURL).toString();
```

Comment:
Detection here, it would be better if we didn't do it by "localhost", but by whether or not an RSA key is set. Can be solved after merge or never, not a big deal.

May also be nice to move "areWorldsConfigured()" to its own function.